chapter_convolutional-neural-networks/channels.md

```diff
@@ -70,21 +76,35 @@ corr2d_multi_in(X, K)
 如下所示，我们实现一个[**计算多个通道的输出的互相关函数**]。
 
 ```{.python .input}
-#@tab all
+#@tab mxnet, pytorch, tensorflow
 def corr2d_multi_in_out(X, K):
     # 迭代“K”的第0个维度，每次都对输入“X”执行互相关运算。
     # 最后将所有结果都叠加在一起
     return d2l.stack([corr2d_multi_in(X, k) for k in K], 0)
 ```
 
+```{.python .input}
+#@tab paddle
+def corr2d_multi_in_out(X, K):
+    # 迭代“K”的第0个维度，每次都对输入“X”执行互相关运算。
+    # 最后将所有结果都叠加在一起
+    return paddle.stack([corr2d_multi_in(X, k) for k in K])
+```
+
 通过将核张量`K`与`K+1`（`K`中每个元素加$1$）和`K+2`连接起来，构造了一个具有$3$个输出通道的卷积核。
 
 ```{.python .input}
-#@tab all
+#@tab mxnet, pytorch, tensorflow
```

Comment:
Consider reusing the code. e.g., we can write a new function called stack in d2l/paddle.py.

Reply:
resolved